st/clitests/awss3api_spec.py

```diff
@@ -1150,6 +1150,8 @@ def get_md5_sum(fname):
 print(parts)
 
 result=AwsTest('Aws cannot complete multipart upload when non-last part is less than 5MB').complete_multipart_upload("seagatebuckettag", "10Mbfile", parts, upload_id).execute_test(negative_case=True).command_should_fail().command_error_should_have("EntityTooSmall")
+#Above test would not abort multipart operation, so need to do it explicitly
+result=AwsTest('Aws abort previous multipart upload').abort_multipart_upload("seagatebuckettag", "10Mbfile", upload_id).execute_test(negative_case=True).command_is_successful()
```

Comment:
As discussed if we don't have this change, ST fails on the very next line, as the next line creates the same multipart upload, thereby failing because, now, we don't abort/delete the multipart after returning EntityTooSmall error.